analyzers/tests/SonarAnalyzer.UnitTest/Rules/AssertionArgsShouldBePassedInCorrectOrderTest.cs

```diff
@@ -33,29 +33,25 @@ public class AssertionArgsShouldBePassedInCorrectOrderTest
         [DataRow("1.1.11")]
         [DataRow(Constants.NuGetLatestVersion)]
         public void AssertionArgsShouldBePassedInCorrectOrder_MsTest(string testFwkVersion) =>
-            OldVerifier.VerifyAnalyzer(
-                @"TestCases\AssertionArgsShouldBePassedInCorrectOrder.MsTest.cs",
-                new AssertionArgsShouldBePassedInCorrectOrder(),
-                NuGetMetadataReference.MSTestTestFramework(testFwkVersion).ToArray());
+            new VerifierBuilder<AssertionArgsShouldBePassedInCorrectOrder>()
```

Comment:
in order to reduce duplication, you can extract the constructor call to a local field and reuse it in the other tests